doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -87,23 +107,66 @@ of the computer.
 
    *
 
-    - :class:`KFold` **(n, k)**
+    - :class:`KFold` **(n_folds, shuffle, random_state)**
+
+    - :class:`LabelKFold` **(n_folds, shuffle, random_state)**
+
+    - :class:`LeaveOneLabelOut` **()**
+
+    - :class:`LeaveOneOut` **()**
+
+   *
+
+    - Split it K folds, train on K-1 and then test on left-out.
+
+    - To ensure that the same label is not in both testing and training sets.
+
+    - Takes a label array to group observations.
+
+    - Leave one observation out.
+
+
+.. list-table::
+
+   *
+
+    - :class:`LeavePLabelOut`  **(p)**
+
+    - :class:`LeavePOut` **(p)**
+
+    - :class:`ShuffleSplit` **()**
+
+    - :class:`StratifiedKFold` **()**
+
+
+   *
+
+    - Leave P labels out.
+
+    - Leave P observations out.
+
+    - Generate train/test indices based on random permutation.
+
+    - It preserves the class ratios/label distribution within each fold.
+
+.. list-table::
+
+   *
 
-    - :class:`StratifiedKFold` **(y, k)**
+    - :class:`StratifiedShuffleSplit`
 
-    - :class:`LeaveOneOut` **(n)**
+    - :class:`PredefinedSplit`
 
-    - :class:`LeaveOneLabelOut` **(labels)**
+    - :class:`LabelShuffleSplit`
 
    *
 
-    - Split it K folds, train on K-1 and then test on left-out
+    - It preserves the class ratios/label distribution within each iteration.
```

Comment:
Maybe "same as shuffle split, while preserving the class distribution within each iteration". Using "label" here is not good.